src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -260,6 +277,48 @@ public function addFluentCommands(Connection $connection, Grammar $grammar)
         }
     }
 
+    /**
+     * Add the alter commands if whenever needed.
+     *
+     * @param  \Illuminate\Database\Connection  $connection
+     * @param  \Illuminate\Database\Schema\Grammars\Grammar  $grammar
+     * @return void
+     */
+    public function addAlterCommands(Connection $connection, Grammar $grammar)
```

Comment:
Step 4. SQLite doesn't support some alter commands, so we need to recreate the table in order to apply changes.
This method walks through the commands and adds "alter commands" whenever needed.
If any alter command has been added, we initiate an "state" for the Blueprint.